vmm/src/memory_manager.rs

```diff
@@ -749,7 +750,7 @@ impl MemoryManager {
             .ok_or(Error::AllocateMmioAddress)?;
 
         #[cfg(not(feature = "tdx"))]
-        let log_dirty = true;
```

Comment:
Is this what is stopping huge pages from being used? What is you just flip the default to false?

Reply:
The default value is still true.